pkg/cmd/grafana-cli/commands/listremote_command.go

```diff
@@ -15,11 +15,10 @@ func listRemoteCommand(c utils.CommandLine) error {
 	}
 
 	for _, p := range plugin.Plugins {
-		plugin := p
-		if len(plugin.Versions) > 0 {
-			ver := latestSupportedVersion(&plugin)
+		if len(p.Versions) > 0 {
```

Comment:
Yeah it was because of the whole Golang and pointer in a loop thing. I have removed the need for pointers across various functions, so this shouldn't be an issue 🤞